doc/source/point-in-time-recovery.rst

```diff
@@ -56,9 +58,15 @@ Run |pbm-restore| and specify the timestamp from the valid range:
 
 .. code-block:: bash
 
-   $ pbm restore --time="2020-07-14T14:27:04"
+   $ pbm restore --time="2020-12-14T14:27:04"
+
+Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of this backup. The time stamp you specify for the restore must meet the following conditions:
+
+* It must be from the time ranges in the PITR section of ``pbm list`` output
+* It must be later than the backup snapshot used for the restore. 
```

Comment:
_If `--base-snapshot` is set. Otherwise, PBM will choose the base on it's own and the user doesn't have think about that - just follow the first condition_

Reply:
Won't this auto-selected snapshot be earlier in time than the specified PITR timestamp?

Reply:
It will. I just meant that if the user doesn't set `--base-snapshot` they shouldn't think about any snapshot. Any time that follows the first condition implicitly has some backup that finished before this time and pbm will choose that backup automatically. So the user doesn't have to worry about any snapshot in such a case.

Maybe
```
If `--base-snapshot` is used the timestamp must be later than the backup snapshot used for the restore.
```

Reply:
Rewritten